### PR TITLE
KLBP-23: Navigation bar and cta

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -1,14 +1,62 @@
...
-      </ul>
+    <aside className="md:bg-white w-full md:w-3/12 lg:w-120 flex-shrink-0 p-6 md:py-12 lg:py-20 px-8 md:shadow-md">
+      {isNavLoaded ? (
+        <div className="pb-4 flex justify-between items-center border-b border-gray-500 md:border-none">
+          <ul className="text-black text-opacity-50 text-2xl">
+            {navigation.map((item, index) => {
+              let itemVisited = ''
+              if (item.done && !item.active) {
+                itemVisited = 'sidebarItemVisited'
+              }
```
Comment (andreir-suciu) [inline src/components/Sidebar.tsx:19]: I think you could use ternary here
`
let itemVisited = item.done && !item.active ? 'sidebarItemVisited' : ''
`

Comment (iuliudascalu) [reply src/components/Sidebar.tsx]: Done.

**Review by andreir-suciu — APPROVED** [review]

File: src/__mock__/breadcrumbMock.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import { IBreadCrumb } from '../interfaces/Breadcrumb'
 
 export const defaultBreadcrumb: IBreadCrumb[] = [
-  { titre: 'Société', active: false },
-  { titre: 'Représentant Légal', active: false },
-  { titre: 'Récapitulatif', active: false },
+  { titre: 'Identité', active: false },
```
Comment (mariuspop86) [inline src/__mock__/breadcrumbMock.ts:4]: Please revert this as these are the lbp values

File: src/__tests__/components/Sidebar.test.tsx [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 test('Renders Sidebar component', () => {
-  intlProviderWrapper({
-    children: <Sidebar />
+  intlRouterProviderWrapper({
+    children: <Sidebar />,
+    initialEntries: [ROUTES.step1],
   })
 
-  const menuItems = ['Société', 'Représentant Légal', 'Récapitulatif']
+  const menuItems = [
```
Comment (mariuspop86) [inline src/__tests__/components/Sidebar.test.tsx:14]: Please revert this as these are the lbp values

File: src/__tests__/pages/template/DefaultTemplate.test.tsx [hunk 4]
```diff
@@ -2,19 +2,28 @@ import React from 'react'
...
       <DefaultTemplate>
         <p>Default template</p>
       </DefaultTemplate>
     ),
+    initialEntries: [ROUTES.step1],
   })
 
   expect(screen.getByText('Default template')).toBeInTheDocument()
-  const menuItems = ['Société', 'Représentant Légal', 'Récapitulatif']
+  const menuItems = [
```
Comment (mariuspop86) [inline src/__tests__/pages/template/DefaultTemplate.test.tsx:19]: Please revert this as these are the lbp values

File: src/components/Sidebar.tsx [hunk 5]
```diff
@@ -1,14 +1,60 @@
...
-        <DynamicFormattedMessage id="sidebar.step1" tag="li" />
-        <DynamicFormattedMessage id="sidebar.step2" tag="li" />
-        <DynamicFormattedMessage id="sidebar.step3" tag="li" />
-      </ul>
+    <aside className="md:bg-white w-full md:w-3/12 lg:w-120 flex-shrink-0 p-6 md:py-12 lg:py-20 px-8 md:shadow-md">
+      {isNavLoaded ? (
+        <div className="pb-4 flex justify-between items-center border-b border-gray-500 md:border-none">
+          <ul className="text-black text-opacity-50 text-2xl">
+            {navigation.map((item, index) => {
+              const itemVisited =
```
Comment (mariuspop86) [inline src/components/Sidebar.tsx:16]: How about if we move this block to it's own component, NavigationItems maybe?

Comment (iuliudascalu) [reply src/components/Sidebar.tsx]: Done.

**Review by mariuspop86 — APPROVED** [review]

**Pull request merged**